app/models/ingredient_transfer.rb

```diff
@@ -0,0 +1,8 @@
+class IngredientTransfer < ApplicationRecord
+belongs_to :source_inventory, class_name: 'Inventory', foreign_key: :source_inventory_id
+belongs_to :target_inventory, class_name: 'Inventory', foreign_key: :target_inventory_id
+belongs_to :ingredient, class_name: 'Ingredient', foreign_key: :ingredient_id
+
+has_many :inventories
+
+end
```

Comment:
Your continual disregard for standardized indentation (as outlined in the Ruby style guide) makes me sad in my soul.

Reply:
I'll go ahead and review that, because I do care about your soul